SafeguardSessions/README.md

```diff
@@ -15,10 +15,81 @@
 
 ## About the project
 
-The aim of One Identity Safeguard Power BI Connector (Power BI Connector) is to provide a solution for customers to visualize their audit data captured by [One Identity Safeguard for Priviledged Sessions] (SPS) in a highly configurable way compared to the on-box reporting system of SPS. The Power BI Connector uses basic authentication with the local login method to connect to SPS, and imports sessions metadata that matches the criteria specified in the Power BI Connector's input parameters. After successful data retrieval, the Power BI Connector returns two tables:
-
-- **Info**: contains information about the data fetching process for debugging purposes, and the version of the Power BI Connector.
-- **Sessions**: contains the actual sessions metadata
+The aim of One Identity Safeguard Power BI Connector (Power BI Connector) is to provide a solution for customers to visualize their audit data captured by [One Identity Safeguard for Priviledged Sessions] (SPS) in a highly configurable way compared to the on-box reporting system of SPS. The Power BI Connector uses basic authentication with the local login method to connect to SPS, and imports sessions metadata that matches the criteria specified in the Power BI Connector's input parameters. After successful data retrieval, the Power BI Connector returns the following two tables.
+
+### Info
+
+This table contains information about the data fetching process for debugging purposes, and the version of the Power BI Connector.
+
+### Sessions
+
+This table contains the actual sessions metadata. The Power BI Connector fetches all sessions from SPS, regardless of their origin or protocol. Currently, the Power BI Connector fetches the following fields of the sessions and nothing more:
```

Comment:
I wanted to be really explicit, but you are right, it is needless to say "regradless of their origin or protocol". I left out the "matching query based constructed from provided field name value pairs." part because we already state "imports sessions metadata that matches the criteria specified in the Power BI Connector's input parameters." in one of the previous sentences.